doc/api/readline.markdown

```diff
@@ -41,6 +41,10 @@ the following values:
 
  - `historySize` - maximum number of history lines retained. Defaults to `30`.
 
+If a `completer` function is supplied, the user will not be able to enter tab
+characters as part of their input. If no `completer` function is supplied, tabs
```

Comment:
I think you're right and barring contrary opinion from someone else, I'll remove that second sentence.